apps/wing-console/console/ui/src/layout/status-bar.tsx

```diff
@@ -65,8 +66,9 @@ export const StatusBar = ({
         </div>
       </div>
       {/*right side*/}
-      <div className="w-full flex space-x-0 justify-end">
+      <div className="w-full flex space-x-1 justify-end">
         <AutoUpdater />
+        <DiscordButton />
```

Comment:
we want users to join our community - the link to discord is too hidden in my opinion